src/cheri_insts.sail

```diff
@@ -726,7 +587,7 @@ union clause ast = CClearTag : (regidx, regidx)
  */
 function clause execute (CClearTag(cd, cs1)) = {
   let cs1_val = C(cs1);
-  C(cd) = {cs1_val with tag=false};
+  C(cd) = invalidateCap(cs1_val);
```

Comment:
Agreed. I changed `invalidateCap` to `clearTag`, since I think we're pretty settled on not legalizing to null. That's as explicit, but still consistent with all the other places where the function is used.